policies/stable-release-updates.md

```diff
@@ -6,8 +6,10 @@ This policy covers allowed changes on stable release branches.
 Definitions
 -----------
 
-A **stable release** is a patch release from an existing supported minor
-release branch.
+A **stable release** is a series beginning with a major or minor release that
+is not a pre-release, and all its updates.
```

Comment:
This sounds more like a stable branch, from which releases are made.

It's a stable release any release not marked as alpha, beta or release candidate?

Reply:
> It's a stable release any release not marked as alpha, beta or release candidate?

For the moment, that's how we've used the term...

> This sounds more like a stable branch, from which releases are made.

That too.  But the way I understand our use of the word is that, for example, OpenSSL 3.0 is a stable release.  Does that just include 3.0.0, or does it include 3.0.x?  The way we've talked about it, the latter seems to apply.

Reply:
I definitely refer to 3.0 or 1.1.1 as a stable release. So the 3.0 stable release means any release from the series 3.0.x. There is some ambiguity in the term "release" here. Sometimes is means a specific release 3.0.1 or 3.0.2 for example. Sometimes it means the set of all releases from a stable branch, e.g. 3.0.x

Reply:
For me 3.0 is a stable branch, the releases in it our 3.0.0, 3.0.1, ...